app/config.go

```diff
@@ -41,8 +41,9 @@ const (
 )
 
 type GCPIMConfig struct {
-	ProjectID string
-	HostImage string
+	ProjectID               string
+	HostImage               string
+	ContainsAcloudInstances bool
```

Comment:
This shouldn't mention "acloud". Create a config entry that simply disables creating and destroying hosts. This could make sense in projects where acloud is not in use, for example if a user's host are created by a different entity and shared with them read-only.

Reply:
done.